### ISSUE TITLE
Updating of critic model in COMA etc

### ISSUE THREAD

**Issue by mitchwenman** [Replicable-MARL/MARLlib#68]
Hi.

I am reading through MARLlib's implementation for a related academic project. In looking at the COMA model implementation, I see functions to update the critic (and actor) models by calling backward() and step() on the gradients. However, I cannot see how and where these are ever called, so it's not clear to me how you are getting RLLib to update the critic weights. Is this done implicitly because cc_rnn subclasses both nn.Module and TorchModel through the parent base_rnn?

Thanks for producing this library as well. It's a mammoth effort :)

**Comment by mitchwenman** [comment]
Disregard. I think I found that RLLib keeps track of parameters needing gradient updates for objects that subclass nn.Module within its TorchV2 models.
